bigquery_etl/cli/dag.py

```diff
@@ -83,7 +90,15 @@ def info(name, dags_config, with_tasks):
 
 
 @dag.command(
-    help="Create a new DAG with name bqetl_<dag_name>, for example: bqetl_search"
+    help="""Create a new DAG with name bqetl_<dag_name>, for example: bqetl_search
+
+    Examples:
+
+    ./bqetl dag create bqetl_core --schedule-interval="0 2 * * *"
+     --owner=example@mozilla.com
+     --description="Tables derived from `core` pings sent by mobile applications."
+     --start-date=2019-07-25
```

Comment:
```suggestion
    ./bqetl dag create bqetl_core --schedule-interval="0 2 * * *" \\
      --owner=example@mozilla.com \\
      --description="Tables derived from `core` pings sent by mobile applications." \\
      --start-date=2019-07-25
```

I think most shells would require `\` for this to work.